frontend/src/scss/global.scss

```diff
@@ -1,49 +1,79 @@
 @import 'typography.scss';
-@import "colors";
+@import 'mixins.scss';
+@import 'colors';
 
 a {
     text-decoration: none;
 }
 
-#title {
-    font-family: $font-header;
-    font-size: 42px;
-    position: relative;
-    padding-bottom: 3rem;
+// Container classes
+
+.blue-container {
+    background: $blue;
+    padding: 2rem 6rem;
+}
+
+.white-container {
+    background: $white;
+    padding: 2rem 6rem;
 }
 
-#white-title {
+// Container title class
+
+.title {
+    display: flex;
+    justify-content: center;
+    padding: 3rem;
+}
+
+// Title classes
+
+.white-title {
+    color: $white;
     font-family: $font-header;
-    font-size: 42px;
-    position: relative;
-    padding-bottom: 3rem;
+    font-size: 2.8rem;
+
+    &::before {
+        @include title-before;
+        background-color: $white;
+    }
 }
-#white-title::before {
-    content: '';
-    position: absolute;
-    bottom: 0;
-    left: 50%;
-    width: 250px;
-    height: 5px;
-    background-color: $white;
-    margin-left: -125px;
-    margin-bottom: 3rem;
+
+.blue-title {
+    color: $blue;
+    font-family: $font-header;
+    font-size: 2.8rem;
+
+    &::before {
+        @include title-before;
+        background-color: $blue;
+    }
```

Comment:
I guess there is no way of nesting blue-title in white title because of &::before, right?